lib/Onnxifi/Base.cpp

```diff
@@ -38,6 +45,16 @@ void Event::wait() {
 onnxStatus Graph::initGraph(const void *onnxModel, size_t onnxModelSize,
                             uint32_t weightCount,
                             const onnxTensorDescriptorV1 *weightDescriptors) {
+  // TODO: support multiple functions here.
+  function_ = backendPtr_->getEE().getModule().createFunction("inference");
+
+  std::unique_ptr<ModelLoader> loader = ModelLoader::parse(
```

Comment:
loader should be a member function of Graph, right?

Reply:
I'll likely need to compile at this point (on initGraph), not sure if loader will stay at that point. We can revisit this later.